thresher/models.py

```diff
@@ -180,6 +203,9 @@ class SubmittedAnswer(models.Model):
     # The highlight group this answer is part of
     highlight_group = models.ForeignKey(HighlightGroup)
 
+    # The id of the user who submitted this answer
+    user_id = models.OneToOneField(User)
```

Comment:
Also needs to be a ForeignKey because a user will almost always have multiple submitted answers.